pylintrc

```diff
@@ -1,327 +1,153 @@
+# ***************************
+# ** DO NOT EDIT THIS FILE **
+# ***************************
+#
+# It is generated by:
+#   $ edx_lint write pylintrc
+#
+#
+#
+#
+#
+#
+#
+#
+# STAY AWAY!
+#
+#
+#
+#
+#
+# SERIOUSLY.
+#
+# ------------------------------
 [MASTER]
-
-# Specify a configuration file.
-#rcfile=
-
-# Python code to execute, usually for sys.path manipulation such as
-# pygtk.require().
-#init-hook=
-
-# Profiled execution.
-profile=no
-
-# Add files or directories to the blacklist. They should be base names, not
-# paths.
-ignore=CVS, migrations
-
-# Pickle collected data for later comparisons.
-persistent=yes
-
-# List of plugins (as comma separated values of python modules names) to load,
-# usually to register additional checkers.
-load-plugins=edx_lint.pylint
-
+profile = no
+ignore = ,migrations
```

Comment:
The tweaks file has "ignore+ = ,migrations" so that if there's an existing ignore setting, it will be appended properly.  As it happens, the master file has an empty ignore setting, so we end up with a leading comma.  It doesn't seem to hurt anything, and I wanted to keep things simple at first.  

The next simplest thing would be to append the string, and then trim leading punctuation.